log/loggers.go

```diff
@@ -129,6 +130,11 @@ type stdLogger struct {
 }
 
 func (sl *stdLogger) Log(lvl int, msg string) {
+	if sl.lInfo.DebugLvl <= 0 {
```

Comment:
A comment there to explain why you're printing although the level is below 0 could help future readers (and me by the occasion)